docs/src/components/home/ReferencesCore.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import CompaniesGrid, { CORE_COMPANIES } from './CompaniesGrid';
+
+const ReferencesCore = () => {
+  return (
+    <Container sx={{ py: { xs: 4, sm: 6, md: 8 } }}>
+      <CompaniesGrid data={CORE_COMPANIES} />
+      <Typography
```

Comment:
I think we're missing declaring this text color on dark mode, which is `grey.500`.